fmriprep/workflows/bold/registration.py

```diff
@@ -449,7 +456,7 @@ def init_bbreg_wf(use_bbr, bold2t1w_dof, omp_nthreads, name='bbreg_wf'):
 
     mri_coreg = pe.Node(
         MRICoregRPT(dof=bold2t1w_dof, sep=[4], ftol=0.0001, linmintol=0.01,
-                    generate_report=not use_bbr),
+                    generate_report=not use_bbr, no_cras0=init_header),
         name='mri_coreg', n_procs=omp_nthreads, mem_gb=5)
```

Comment:
What purpose is `mri_coreg` serving now? Should we just remove the node if we're using the header?

Reply:
This is a good question--I don't have experience using `mri_coreg` myself, so I'm not sure how valuable it is in this situation. Regardless, skipping mri_coreg was also my intuition when I first started--i.e., that the mri_coreg node should not be linked up if `init_header` is true. However, this turned out to be substantially harder to get working--my edits removing mri_coreg resulted in a successful registration, but they disrupted downstream nodes that I'm not familiar with, so I opted to go for this (minimal) change to the existing pipeline.